assets/css/modules/_buy-sell.scss

```diff
@@ -1,6 +1,6 @@
 .buy-sell {
   position: relative;
-  padding: 20px 30px 0 30px;
+  padding: 26px 30px 0 30px;
```

Comment:
Let's leave this as 20px, otherwise Buy Bitcoin on Coinify looks too spaced out.

<img width="567" alt="screen shot 2017-01-17 at 8 39 31 pm" src="https://cloud.githubusercontent.com/assets/5640772/22047519/96d8860e-dcf5-11e6-99d4-ae3b395e8e03.png">